libpod/util.go

```diff
@@ -76,3 +77,13 @@ func GetPolicyContext(path string) (*signature.PolicyContext, error) {
 	}
 	return signature.NewPolicyContext(policy)
 }
+
+const alpha = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+func GenerateRandomString(n int) string {
```

Comment:
Instead of doing this, you can use the existing random ID generation we import from Docker, and just take the first n characters of the output

Reply:
pointer?

Reply:
Most notably used in newContainer() in container.go